### PR TITLE
add negative tests for multiTable

### PR DESCRIPTION
**Pull request by andriidovhan**

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy [hunk 1]
```diff
@@ -1145,6 +1147,70 @@ mode with existing flows and hold flows of different table-mode types"() {
         northbound.deleteLinkProps(northbound.getAllLinkProps())
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2932")
+    def "System does not allow ot enable the multiTable mode on an unsupported switch"(){
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy]: topology dependent

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy [hunk 2]
```diff
@@ -1145,6 +1147,70 @@ mode with existing flows and hold flows of different table-mode types"() {
...
+        })
+
+        then: "Human readable error is returned"
+        def exc = thrown(HttpClientErrorException)
+        exc.rawStatusCode == 404
+        //TODO(andriidovhan) fix error message when 2932 is fixed
+        exc.responseBodyAsString.to(MessageError).errorMessage == "Switch doesn't support multiTable"
+    }
+
+    def "System connects a new switch with disabled multiTable mode when the switch does not support that mode"() {
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy]: same

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy [hunk 3]
```diff
@@ -1145,6 +1147,70 @@ mode with existing flows and hold flows of different table-mode types"() {
...
+        given: "Unsupported switch"
+        def sw = topology.activeSwitches.find { !it.features.contains(SwitchFeature.MULTI_TABLE) }
+        assumeTrue("Unable to find required switch", sw as boolean)
+        def initConf = northbound.getKildaConfiguration()
+        !initConf.useMultiTable && northbound.updateKildaConfiguration(northbound.getKildaConfiguration().tap {
+            it.useMultiTable = true
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy]: It is an important step of the test case, I believe state of global config should be explicitly described as a precondition

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy (hunk 1 above)
Comment (andriidovhan) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy]: agree

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy (hunk 2 above)
Comment (andriidovhan) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy]: added

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy (hunk 3 above)
Comment (andriidovhan) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy]: added

**Pull request merged**